launch_only_docker.sh

```diff
@@ -1,3 +1,4 @@
 #Bind the host machine's urc-2023 root folder to the container's target path.
 #Launch the "trickfireimage" image by giving shell control over to the container
-docker run --mount type=bind,source="$(pwd)",target="/home/trickfire/urc-2023" -it --privileged --network=host trickfireimage
\ No newline at end of file
+#Launch the robot code
+docker run --mount type=bind,source="$(pwd)",target="/home/trickfire/urc-2023" -it --privileged --network=host trickfireimage /home/trickfire/urc-2023/launch.sh
```

Comment:
While this does run the code, it does not ./build.sh the code. Just leave it at /home/trickfire/urc-2023 for now I think.